iscsi/iscsi_test.go

```diff
@@ -98,78 +101,81 @@ func (s *TestSuite) TestFlow(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(tmptid, Equals, 1)
 
-	exists = IsTargetLoggedIn(s.localIP, t, s.ne)
+	exists = IsTargetLoggedIn(s.localIP, t, s.nsexec)
 	c.Assert(exists, Equals, false)
 
-	exists = IsTargetLoggedIn("", t, s.ne)
+	exists = IsTargetLoggedIn("", t, s.nsexec)
 	c.Assert(exists, Equals, false)
 
-	err = DiscoverTarget(s.localIP, t, s.ne)
+	err = DiscoverTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, IsNil)
 
-	exists = IsTargetDiscovered(s.localIP, t, s.ne)
+	exists = IsTargetDiscovered(s.localIP, t, s.nsexec)
 	c.Assert(exists, Equals, true)
 
-	err = DeleteDiscoveredTarget(s.localIP, t, s.ne)
+	err = DeleteDiscoveredTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, IsNil)
 
-	exists = IsTargetDiscovered(s.localIP, t, s.ne)
+	exists = IsTargetDiscovered(s.localIP, t, s.nsexec)
 	c.Assert(exists, Equals, false)
 
-	err = DeleteDiscoveredTarget(s.localIP, t, s.ne)
+	err = DeleteDiscoveredTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, NotNil)
 
-	err = LoginTarget(s.localIP, t, s.ne)
+	err = LoginTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, NotNil)
 
-	err = DiscoverTarget(s.localIP, t, s.ne)
+	err = DiscoverTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, IsNil)
 
-	exists = IsTargetDiscovered(s.localIP, t, s.ne)
+	exists = IsTargetDiscovered(s.localIP, t, s.nsexec)
 	c.Assert(exists, Equals, true)
 
-	err = LoginTarget(s.localIP, t, s.ne)
+	err = LoginTarget(s.localIP, t, s.nsexec)
 	c.Assert(err, IsNil)
 
-	exists = IsTargetLoggedIn(s.localIP, t, s.ne)
+	exists = IsTargetLoggedIn(s.localIP, t, s.nsexec)
 	c.Assert(exists, Equals, true)
 
-	exists = IsTargetLoggedIn("", t, s.ne)
+	exists = IsTargetLoggedIn("", t, s.nsexec)
 	c.Assert(exists, Equals, true)
 
-	dev, err := GetDevice(s.localIP, t, lun, s.ne)
+	ne, err := util.NewNamespaceExecutor("/host/proc/1/ns/")
+	c.Assert(err, IsNil)
+
+	dev, err := GetDevice(s.localIP, t, lun, s.nsexec, ne)
```

Comment:
In this case, what's difference between `s.nsexec` and `ne`?

Reply:
This change is temporary and has been subsequently removed in a later commit, https://github.com/longhorn/go-iscsi-helper/pull/69/commits/8f90cd73ec6c8f15eb8b54914d03c4c3731e3ed8#diff-a250613c0bbb5af7a178f183cbf68f9f217bdff1c6b09c61dfca5c36ede34382L146